librz/core/cmd_help.c

```diff
@@ -876,7 +876,6 @@ RZ_IPI int rz_cmd_help(void *data, const char *input) {
 			pj_ks(pj, "arch", RZ_SYS_ARCH);
 			pj_ks(pj, "os", RZ_SYS_OS);
 			pj_ki(pj, "bits", RZ_SYS_BITS);
-			pj_ks(pj, "tap", RZ_GITTAP);
```

Comment:
Is this the git version stuff that you removed?

Reply:
Yes. https://github.com/rizinorg/rizin/pull/433/files#diff-30d8f6be6320feeacf686be94f48c70869b52630e01ea625f0f15adc0d57c3e4L65 It essentially took the last git tag. However, as said, I don't know why we would want that. I think we only care about RZ_VERSION.

ANyway, given how big this diff is maybe it's better if I extract in a separate PR this commit so you can review it better.

Reply:
Ok look at https://github.com/rizinorg/rizin/pull/480 first.